src/Microsoft.Health.SqlServer/Features/Schema/Manager/ISchemaManagerDataStore.cs

```diff
@@ -15,8 +15,9 @@ public interface ISchemaManagerDataStore : IReadOnlySchemaManagerDataStore
         /// </summary>
         /// <param name="script">The script to execute</param>
         /// <param name="version">The version to update its status</param>
+        /// <param name="applyFullSchemaSnapshot">A flag to know if its a full schema snapshot</param>
         /// <param name="cancellationToken">A cancellation token</param>
-        Task ExecuteScriptAndCompleteSchemaVersionAsync(string script, int version, CancellationToken cancellationToken);
+        Task ExecuteScriptAndCompleteSchemaVersionAsync(string script, int version, bool applyFullSchemaSnapshot, CancellationToken cancellationToken);
```

Comment:
nit: applyFullSchemaSnapshot -> isDiffScript?
(Since we need to do something extra if diff script)

Reply:
at other places we are using this flag with the same name so may be to be consistent ?